ginivision/src/main/java/net/gini/android/vision/analysis/AnalysisFragmentImpl.java

```diff
@@ -110,15 +133,111 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle sa
         return view;
     }
 
+    private Handler mHandler = new Handler(Looper.getMainLooper());
+
     public void onDestroy() {
         mPhoto = null;
         stopScanAnimation();
     }
 
     public void onStart() {
+        showHints();
+    }
+
+    private void showHints() {
+
+        final int containerHeight = getContainerHeight();
+        mHints = generateRandomHintsList();
+
+        mRunnable = new Runnable() {
+            @Override
+            public void run() {
+                mHintAnimation = getSlideDownAnimation(containerHeight);
+                mHintAnimation.start();
+            }
+        };
+        mHandler.postDelayed(mRunnable, HINT_START_DELAY);
+    }
+
+    @NonNull
+    private ViewPropertyAnimatorCompat getSlideDownAnimation(final int containerHeight) {
+        return ViewCompat.animate(mHintContainer)
```

Comment:
You could try using the support TransitionManager. We wouldn't need to get the container height.